src/modules/configurator/components/const/toastify.ts

```diff
@@ -0,0 +1,18 @@
+import { ToastPosition, Theme } from "react-toastify"
+import "react-toastify/dist/ReactToastify.css";
+
+interface ToastifyProps {
+    position: ToastPosition;
+    autoClose: number;
+    hideProgressBar: boolean;
+    closeOnClick: boolean;
+    theme: Theme;
+}
```

Comment:
There is probably this type that you can import from package

Reply:
Imported ToastOptions type from package.